tests/scripts/arm32_ci_script.sh

```diff
@@ -227,7 +227,7 @@ function cross_build_coreclr {
     fi
 
     #Cross building for emulator rootfs
-    ROOTFS_DIR="$__ARMEmulRootfs" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build.sh $__buildArch cross $__verboseFlag $__skipMscorlib clang3.5 $__buildConfig -rebuild
+    ROOTFS_DIR="$__ARMEmulRootfs" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build.sh $__buildArch cross $__verboseFlag $__skipMscorlib clang3.9 $__buildConfig -rebuild
```

Comment:
How about leave this code as is? 
I'm afraid this command is not executed in Docker, but in host environment where clang 3.9 is not available.